src/extension.ts

```diff
@@ -68,23 +68,21 @@ export async function activate(ctx: vscode.ExtensionContext) {
   controller.createRunProfile("Meson run test", vscode.TestRunProfileKind.Run, (request, token) => testRunHandler(controller, request, token), true)
   ctx.subscriptions.push(controller);
 
-  let mesonTasks: Thenable<vscode.Task[]> | undefined = undefined;
+  let mesonTasks: Thenable<vscode.Task[]> | null = null;
```

Comment:
OOC, what difference does it make to use null instead of undefined? I copied that line verbatim from vscode doc...

Reply:
Sorry I didn't really mean to commit that, was going to suggest some cleanups separately (after turning on `strictNullChecks`, but didn't really have to time to finish the job).

More because of the assigning `undefined` later which I think is considered unpleasant. Crazy language design!